bundle/src/main/java/com/adobe/acs/commons/indesign/dynamicdeckdynamo/models/DynamicDeckInitiatorPageModel.java

```diff
@@ -0,0 +1,154 @@
+package com.adobe.acs.commons.indesign.dynamicdeckdynamo.models;
+
+import com.adobe.acs.commons.mcp.form.*;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.models.annotations.DefaultInjectionStrategy;
+import org.apache.sling.models.annotations.Model;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+@Model(
+        adaptables = {Resource.class, SlingHttpServletRequest.class},
+        defaultInjectionStrategy = DefaultInjectionStrategy.OPTIONAL)
+public class DynamicDeckInitiatorPageModel extends GeneratedDialog {
+
+    @Inject
```

Comment:
Can we use an explicit Injector here? Ex. `@ValueMapValue` if it comes from the resource property?